packages/code-migrate/src/loadUserMigrationFile.ts

```diff
@@ -16,7 +16,8 @@ export const loadUserMigrationFile = async (
   return new Promise((resolve) => {
     // Load user's migration file
     const migrate: Migrate = async (title, fn) => {
-      migration.events.emit('migration-start', { title, migration });
+      migration.title = title;
```

Comment:
If the title is on the migration (which makes sense) we can reuse it from the markdown report